development/idp-local/metadata/saml20-idp-hosted.php

```diff
@@ -37,8 +41,17 @@
             'dateFormat' => 'Y-m-d',
             'loggerClass' => Psr3StdOutLogger::class,
         ],
+        30 => [
```

Comment:
Noting that `30` may be already taken by `core:LanguageAdaptor`

Reply:
Must they be unique in different metadata files? I honestly don't know, but my suspicion is that it doesn't matter. We use `30` for "profilereview" in production, I think.

Reply:
It really depends on how they merge the arrays. I suspect it doesn't matter either, only reason I noticed was because of the big config file that had some authprocs in it.